NavigationCamera#onStart NullPointerException

Run into the following 💥 while testing the test app ([`EndNavigationActivity`](https://github.com/mapbox/mapbox-navigation-android/blob/master/app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/EndNavigationActivity.java)):

```
07-04 17:41:05.930 9992-9992/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.mapbox.services.android.navigation.testapp, PID: 9992
    java.lang.RuntimeException: Unable to resume activity {com.mapbox.services.android.navigation.testapp/com.mapbox.services.android.navigation.testapp.activity.navigationui.EndNavigationActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.mapbox.services.android.navigation.v5.navigation.MapboxNavigation.addProgressChangeListener(com.mapbox.services.android.navigation.v5.routeprogress.ProgressChangeListener)' on a null object reference
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3103)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3134)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1388)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.mapbox.services.android.navigation.v5.navigation.MapboxNavigation.addProgressChangeListener(com.mapbox.services.android.navigation.v5.routeprogress.ProgressChangeListener)' on a null object reference
        at com.mapbox.services.android.navigation.ui.v5.camera.NavigationCamera.onStart(NavigationCamera.java:178)
        at com.mapbox.services.android.navigation.ui.v5.map.NavigationMapboxMap.onStart(NavigationMapboxMap.java:141)
        at com.mapbox.services.android.navigation.ui.v5.NavigationView.onStart(NavigationView.java:170)
        at com.mapbox.services.android.navigation.testapp.activity.navigationui.EndNavigationActivity.onStart(EndNavigationActivity.java:201)
        at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1237)
        at android.app.Activity.performStart(Activity.java:6253)
        at android.app.Activity.performRestart(Activity.java:6299)
        at android.app.Activity.performResume(Activity.java:6304)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3092)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3134) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1388) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:148) 
        at android.app.ActivityThread.main(ActivityThread.java:5417) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

👀 

![navigation_camera_on_start_npe](https://user-images.githubusercontent.com/1668582/42286292-ab0bd674-7fb2-11e8-84f7-3e3f62f301fe.gif)

--- a/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/camera/NavigationCamera.java
+++ b/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/camera/NavigationCamera.java
@@ -175,7 +175,9 @@ public class NavigationCamera implements LifecycleObserver {
    */
   @OnLifecycleEvent(Lifecycle.Event.ON_START)
   public void onStart() {
-    navigation.addProgressChangeListener(progressChangeListener);
+    if (navigation != null) {
+      navigation.addProgressChangeListener(progressChangeListener);
+    }
   }
 
   /**

